spec/lib/droplet_kit/kubernetes_resource_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+require 'pry'
+
+RSpec.describe DropletKit::KubernetesResource do
+  subject(:resource) { described_class.new(connection: connection) }
+  let(:kubernetes_node_pool_attributes) { DropletKit::KubernetesNodePool.new.attributes }
+  include_context 'resources'
+
+  describe "cluster_node_pools" do
+    it 'returns the node_pools for a cluster' do
+      cluster_id = 42
```

Comment:
magic number! maybe set the cluster id to `1`? unless you're a big jackie robinson fan and want that 42. :)

Reply:
https://en.wikipedia.org/wiki/42_(number)

Reply:
Although I've never read it, i've always used that number haha. Maybe 23 is better for MJ and LBJ

Reply:
haha who spends that much time coming up with a wiki for a number. so many interesting facts in there though.